### ISSUE TITLE
Better KtLintParseException error description

### ISSUE THREAD

**Issue by avdyushin** [jeremymailen/kotlinter-gradle#333]
In case if code (somehow) has compilation issues (e.g. after merge conflicts), running `lintKotlin` will give this error:

```
lint worker execution error
com.pinterest.ktlint.core.api.KtLintParseException: 17:5 Expecting member declaration
	at com.pinterest.ktlint.core.internal.RuleExecutionContext$Companion.createRuleExecutionContext$ktlint_core(RuleExecutionContext.kt:137)
...
```

Which doesn't clear, what file cause this issue.

**Comment by jeremymailen** [comment]
Yes, good point. Probably we should catch the error from running the ktlint library and wrap it in a better message indicating which file.
